spec/models/project_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Project, type: :model do
+  it { should have_db_column(:title).of_type(:string) }
+  it { should have_db_column(:description).of_type(:string) }
+  it { should have_db_column(:active).of_type(:boolean) }
+  it { should have_db_column(:project_type).of_type(:integer) }
+  it { should have_db_column(:user_id).of_type(:integer) }
+  it { should have_db_column(:started_at).of_type(:datetime) }
+  it { should have_db_column(:close_at).of_type(:datetime) }
+  it { should belong_to(:user) }
+
+  it "valid if title is available" do
```

Comment:
What you are trying to test here ?

Reply:
on 'title' has a validation of null false.

Reply:
remove this.